src/model/data.types.model.ts

```diff
@@ -83,3 +83,6 @@ export const EntityNumberKey = "entity_number";
 export const UpdateModelKey = "update";
 export const DoYouWantToRemoveKey = "do_you_want_to_remove";
 export const AnyTrustsInvolvedKey = "any_trusts_involved";
+
+// Remove Journey
+export const IsListedAsPropertyOwnerKey = "listed_as_property_owner";
```

Comment:
can you add is_ to the "listed_as_property_owner", so it matches the IsListedAsPropertyOwnerKey and then it will be consistent with 
export const IsSecureRegisterKey = "is_secure_register";

Reply:
the "disposed_all_land" is the name of the html element, not the field in our model. I'd prefer to have 
export const IsListedAsPropertyOwnerKey = "is_listed_as_property_owner";
instead of 
export const IsListedAsPropertyOwnerKey = "listed_as_property_owner";
on the model.
Any preference @dlloyd1-ch @mwestacott ?

Reply:
Made this further update